code/builders/builder-webpack5/templates/virtualModuleModernEntry.js

```diff
@@ -0,0 +1,34 @@
+import { createBrowserChannel } from 'storybook/internal/channels';
+import { PreviewWeb, addons, composeConfigs } from 'storybook/internal/preview-api';
+
+import { global } from '@storybook/global';
+
+import { importFn } from './{{storiesFilename}}';
+
+const getProjectAnnotations = () =>
+  composeConfigs(['{{previewAnnotations}}'].map((entry) => require(entry)));
```

Comment:
doesn't this have too many quotes?

```suggestion
  composeConfigs([{{previewAnnotations}}].map((entry) => require(entry)));
```

Reply:
which will make the syntax invalid, so it will probably break the require. I guess you need to change the `replaceAll` call instead.

Reply:
or, just inline the whole file content in the mapper, remvoing the need for the require call.

Reply:
Hmmm. I did run the storybook, but it isn't using this code path, probably 🤕

Reply:
yes that's a fair point.
I feel like using comments for replacements throughout the file would be the cleanest maybe?
`template.replaceAll('/*PREVIEW_ANNOTATIONS*/', previewAnnotations)`

alternatively, a change that is closer to what you already have, would be to just change the replacer to `.replaceAll("'{{previewAnnotations}}'", ...)`